docs/docs/user-guide/cli-options.md

```diff
@@ -50,6 +50,11 @@ Options:
                                             e-page-application crawling or when
                                             different hashtags load dynamic cont
                                             ent
+      --selectLinks                         one or more selectors for extracting
+                                             links, in the format [css selector]
+                                            ->[property to use],[css selector]->
+                                            @[attribute to use]
+                                            [array] [default: ["a[href]->href"]]
```

Comment:
This is a good reference but I feel like this is both an important feature and one where the syntax isn't the most straightforward, so I'd suggest we add a brief section in the Commonly-Used Options page of the docs with an example or two?